utils/prepare_all_artifacts.py

```diff
@@ -28,137 +28,158 @@ def import_module_from_file(file_path):
     return module
 
 
-def prepare_artifacts_for_prepare_files(prepare_files):
-    failed_prepare_files = []
-    prepare_exceptions = []
-    for i, file in enumerate(prepare_files):
-        logger.info("*" * 100)
-        logger.info(f"* {i}/{len(prepare_files)}: {file}")
-        logger.info("*")
-        try:
-            import_module_from_file(file)
-        except Exception as e:
-            logger.info(f"Failed to prepare: {file}")
-            failed_prepare_files.append(file)
-            prepare_exceptions.append(e)
-
-    return failed_prepare_files, prepare_exceptions
-
+# flake8: noqa: C901
+def main():
+    catalog_dir = constants.catalog_dir
+    catalog_back_dir = catalog_dir + "_back"
 
-def prepare_all_catalog_artifacts(catalog_dir):
     os.environ["UNITXT_USE_ONLY_LOCAL_CATALOGS"] = "True"
     os.environ["UNITXT_TEST_CARD_DISABLE"] = "True"
     os.environ["UNITXT_TEST_METRIC_DISABLE"] = "True"
     os.environ["UNITXT_SKIP_ARTIFACTS_PREPARE_AND_VERIFY"] = "True"
```

Comment:
Is `os.environ["UNITXT_SKIP_ARTIFACTS_PREPARE_AND_VERIFY"]` used anywhere in unitxt?